platforms/documentation/docs/src/docs/userguide/dep-man/declaring-repositories/supported_repository_types.adoc

```diff
@@ -0,0 +1,160 @@
+
+Gradle uses the same logic as Maven to determine the location of your local Maven cache.
+It prioritizes the `settings.xml` in the user’s home directory,`<user_home>/.m2`, over the one in `__M2_HOME__/conf`.
+If no `settings.xml` is found, Gradle uses the default location at `<user_home>/.m2/repository`.
+
+[[sec:ivy_repositories]]
+== Ivy repositories
+
+Organizations may choose to host dependencies in an in-house Ivy repository.
+Gradle can declare Ivy repositories by specifying their URL:
+
+[[sub:defining_an_ivy_repository_with_a_standard_layout]]
+=== Standard layout Ivy repository
+
+To declare an Ivy repository with the standard layout, simply specify the URL:
+
+====
+include::sample[dir="snippets/artifacts/defineRepository/kotlin",files="build.gradle.kts[tags=ivy-repo]"]
+include::sample[dir="snippets/artifacts/defineRepository/groovy",files="build.gradle[tags=ivy-repo]"]
+====
+
+[[sub:defining_a_named_layout_for_an_ivy_repository]]
+=== Named layout Ivy repository
+
+You can specify that your repository follows the Ivy or Maven default layout:
+
+====
+include::sample[dir="snippets/artifacts/defineRepository/kotlin",files="build.gradle.kts[tags=ivy-repo-with-maven-layout]"]
+include::sample[dir="snippets/artifacts/defineRepository/groovy",files="build.gradle[tags=ivy-repo-with-maven-layout]"]
+====
+
+Valid named layout values are `gradle` (default), `maven`, and `ivy`.
+Refer to link:{groovyDslPath}/org.gradle.api.artifacts.repositories.IvyArtifactRepository.html#org.gradle.api.artifacts.repositories.IvyArtifactRepository:layout(java.lang.String)[IvyArtifactRepository.layout(java.lang.String)] in the API documentation for more details.
+
+[[sub:defining_custom_pattern_layout_for_an_ivy_repository]]
+=== Custom pattern layout Ivy repository
+
+To define an Ivy repository with a non-standard layout, you can set up a pattern layout:
+
+====
+include::sample[dir="snippets/artifacts/defineRepository/kotlin",files="build.gradle.kts[tags=ivy-repo-with-pattern-layout]"]
+include::sample[dir="snippets/artifacts/defineRepository/groovy",files="build.gradle[tags=ivy-repo-with-pattern-layout]"]
+====
+
+For an Ivy repository that fetches Ivy files and artifacts from different locations, define separate patterns:
+
+====
+include::sample[dir="snippets/artifacts/defineRepository/kotlin",files="build.gradle.kts[tags=ivy-repo-with-custom-pattern]"]
+include::sample[dir="snippets/artifacts/defineRepository/groovy",files="build.gradle[tags=ivy-repo-with-custom-pattern]"]
+====
+
+Optionally, you can enable Maven-style layout for the 'organisation' part, with forward slashes replacing dots:
+
+====
+include::sample[dir="snippets/artifacts/defineRepository/kotlin",files="build.gradle.kts[tags=ivy-repo-with-m2compatible-layout]"]
+include::sample[dir="snippets/artifacts/defineRepository/groovy",files="build.gradle[tags=ivy-repo-with-m2compatible-layout]"]
+====
+
+[[sub:accessing_secured_ivy_repositories]]
+=== Authenticated Ivy repository
```

Comment:
I find it odd that we link to another page for maven authentication but include ivy authentication on this page

Reply:
Everything has been moved to the supported_repo_types page.